src/components/Editor/Tabs.css

```diff
@@ -70,49 +73,45 @@
 }
 
 .source-tab .img.prettyPrint,
-.source-tab .source-icon.blackBox {
+.source-tab .img.blackBox {
   height: 12px;
   width: 12px;
   align-self: center;
 }
 
 .source-tab .img.react {
-  mask: url(/images/react.svg) no-repeat;
-  mask-size: 100%;
-  height: 14px;
-  width: 14px;
   background: var(--theme-highlight-bluegrey);
-  top: 0;
 }
 
 .source-tab .filename {
+  display: block;
   white-space: nowrap;
   text-overflow: ellipsis;
   overflow: hidden;
-  padding: 0 4px;
-  align-self: center;
-  margin-bottom: 1px;
+  padding-inline-end: 2px;
 }
 
 .source-tab .filename span {
```

Comment:
Is this style still necessary? Couldn't find a case where the .filename div has a span inside

Reply:
I think it may happen if you have two files with different paths but the same name?

```
some/thing/index.js
other/stuff/index.js
```

Not sure.

Reply:
Yep, can confirm that. We do need the span styles.